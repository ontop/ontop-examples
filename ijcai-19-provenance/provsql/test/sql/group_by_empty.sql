\set ECHO none
SET search_path TO public,provsql;

CREATE TABLE group_by_empty_result AS
  SELECT provenance()
  FROM personnel
  GROUP BY ();

SELECT remove_provenance('group_by_empty_result');
SELECT security(provenance,'personnel_level') FROM group_by_empty_result;
