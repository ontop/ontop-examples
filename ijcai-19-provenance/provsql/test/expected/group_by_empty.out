\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

   security   
--------------
 unclassified
(1 row)

